v2/tools/generator/internal/codegen/testdata/ArmResource/Arm_oneof_resource_conversion_on_arm_type_only_azure.golden

```diff
@@ -616,7 +616,7 @@ func (properties *Properties) ConvertToARM(resolved genruntime.ConvertToARMResol
 		if err != nil {
 			return nil, err
 		}
-		foo := fooARM.(FooARM)
+		foo := *fooARM.(*FooARM)
```

Comment:
I wonder if we might be able to simplify things to return the expected type directly using generics - in a future PR.